src/main/webapp/js/instructorFeedbackResults.js

```diff
@@ -209,14 +209,21 @@ function toggleCollapse(e, pans) {
     var panels = pans || $('div.panel-collapse');
 
     if ($(e).html().trim().startsWith(expand)) {
-        isExpandingAll = true;
-        var i = 0;
-        for (var idx = 0; idx < panels.length; idx++) {
-            if ($(panels[idx]).attr('class').indexOf('in') === -1) {
-                setTimeout(showSingleCollapse, 50 * i, panels[idx]);
-                i++;
+        $('.ajax-response-auto').click();
```

Comment:
It will not cause any problems if some pages do not have anything selected by ```.ajax-response-auto``` as "clicking on nothing" does nothing

It does create long term maintainance issues when people come around and start asking what it is for, like now :D